lms/djangoapps/grades/new/subsection_grade.py

```diff
@@ -39,22 +41,20 @@ def scores(self):
         """
         List of all problem scores in the subsection.
         """
-        return [score for score, _ in self.locations_to_weighted_scores.itervalues()]
+        scores = [score for score, _ in self.locations_to_weighted_scores.itervalues()]
+        log.info(u"Persistent Grades: calculated scores property for subsection {0}, user {1}".format(self.location))
+        return scores
```

Comment:
Nit: now that we're no longer logging scores, does `scores = [...]` and `return scores` need to be 2 separate lines, or could we just `return [...]` after logging the message?